File: LC/lc26/lc26.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        if nums == []:
            return 0
        p = 0
        for i in range(1, len(nums)):
            if nums[i] == nums[p]:
                continue
            p += 1
            nums[p] = nums[i]
        return p+1

File: LC/lc26/lc26.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
  let p = 0, last = -100000;
  for (i = 0; i < nums.length; i++) {
      if (nums[i] != last){
          nums[p] = nums[i];
          p += 1;
          last = nums[i];
      }
  }
  nums.splice(p, nums.length - p);
};